### PR TITLE
Supports Grakn Core 1.8

### LINKED ISSUE #290: Support Grakn 1.8

### PR DESCRIPTION
**Pull request by sorsaffari**
## What is the goal of this PR?
This PR introduces the changes required to support Grakn 1.8. Addinitially, users are now notified via an error notification, when they try to explain an inferred concept, whose corresponding rule definition contains at least one unassigned relation within the rule's `when` body.

This PR also marks the `1.3.0` release of Workbase.

## What are the changes implemented in this PR?
resolves #290

Following graknlabs/client-nodejs@b85668f
- added query option `{ explain: true }`
- `dataType` to `valueType`
- `datatype` to `value`
- `date` to `datetime`
- removing all references to `isImplicit`
- bump `client-nodejs` and `grakn-core` dependencies

### CODE REVIEW

File: dependencies/graknlabs/dependencies.bzl [hunk 1]
```diff
@@ -29,12 +29,12 @@ def graknlabs_grakn_core():
     git_repository(
         name = "graknlabs_grakn_core",
         remote = "https://github.com/graknlabs/grakn",
-        tag = "1.7.2", # sync-marker: do not remove this comment, this is used for sync-dependencies by @graknlabs_grakn_core
+        commit = "8aa8721caf9a83ed7651e2612686f7738cd323e7", # sync-marker: do not remove this comment, this is used for sync-dependencies by @graknlabs_grakn_core
```
Comment (flyingsilverfin) [inline dependencies/graknlabs/dependencies.bzl:32]: this is now released! 1.8.0

File: src/renderer/components/Visualiser/store/actions.js [hunk 2]
```diff
@@ -223,42 +223,53 @@ export default {
...
-        const data = await CDB.buildInstances(explanationResult);
-        const rpData = await CDB.buildRPInstances(explanationResult, data, false, graknTx);
-        data.nodes.push(...rpData.nodes);
-        data.edges.push(...rpData.edges);
-
-        state.visFacade.addToCanvas(data);
-        commit('updateCanvasData');
-        const nodesWithAttributes = await computeAttributes(data.nodes, graknTx);
+      const when = await (await explanation.getRule()).getWhen();
+      const isRelUnassigned = !when.match(/(\$[^\s]*|;|{)(\s*?\(.*?\))/)[1].includes('$');
```
Comment (flyingsilverfin) [inline src/renderer/components/Visualiser/store/actions.js:229]: lol nice - can you write a unit test for this?

File: dependencies/graknlabs/dependencies.bzl (hunk 1 above)
Comment (sorsaffari) [reply dependencies/graknlabs/dependencies.bzl]: addressed in https://github.com/graknlabs/workbase/pull/298/commits/5e7cb8a458fabb8f8106071aa7ca1417e38417c3

File: src/renderer/components/Visualiser/store/actions.js (hunk 2 above)
Comment (sorsaffari) [reply src/renderer/components/Visualiser/store/actions.js]: thanks for asking for the test. turned out the implementation wasn't correct.

resolved in https://github.com/graknlabs/workbase/pull/298/commits/abf1d365c6d5789cb1b2440dda60a706361ebd6e

Comment (flyingsilverfin) [reply src/renderer/components/Visualiser/store/actions.js]: excellente

**Comment by flyingsilverfin** [comment]
are we going to release the new workbase as 1.3?

**Pull request merged**